### DISCUSSION TITLE
Translations?

### DISCUSSION THREAD

**Discussion by ivanovd** [cioraneanu/firefly-pico#145] [Ideas]
Since Firefly III supports a number of languages, wouldn't it be a natural continuation of that so that the Pico is translated as well? I am sure that many contributors will help with the translation.

**Comment by cioraneanu** [comment]
I don't have anything against it but i don't find it that exciting.
There is an issue with the same proposal here #78.

If anyone wants to take a swing at it pull requests are always welcomed. 😁

**Comment by cioraneanu** [comment]
This is now implemented 😇
